### ISSUE TITLE
Copy link not working on android 10

### ISSUE THREAD

**Issue by thamaraiselvam** [thepracticaldev/dev.to#7109]
Copy button on article social sharing not working on android latest Android app. 

1.go to any article
2. click share icon
3. click  copy icon

it's not copied anything.

Android version 10

App version 1.2.1

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by cmgorton** [comment]
@fdoxyz should this go in the https://github.com/forem/DEV-Android repo?

**Comment by fdoxyz** [comment]
That's right, I can transfer the issue and follow up over there. Thanks for bringing this up!
